docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Gatsby Gazette: The August Edition"
+date: 2019-08-06
+author: Sidhartha Chatterjee
+excerpt: "Welcome to the new and improved Gatsby Gazette"
+tags:
+  - gazette
+---
+
+Remember when I said I'll see you folks next week with a jam packed Gazette? It's been 3 months! 😱 If it's any consolation though, this one is a jam packed edition.
+
+![Meme joking about how I said I'll be back in a week and readers are now skeletons because 3 months passed](./meme.jpg)
```

Comment:
Note sure about the skeleton in this - I don't think I'd count it as family friendly so we should probably remove it.

Reply:
Good point! Replaced with more skeleton less meme 😛